File: Majority Element - E /sol.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int,int> hmap = {};
        for (int i = 0; i < nums.size(); i++) {
            hmap[nums[i]] += 1;
            if (hmap[nums[i]] > int(nums.size()/2)) {
                return nums[i];
            }
        }  
        return -1;        
    }
};

File: Top K Frequent Elements - M/sol.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hmap = defaultdict(int)
        for i in range(len(nums)):
            hmap[nums[i]] += 1
        res = []
        for i in range(k):
            key_with_max_value = max(hmap, key=hmap.get)
            res.append(key_with_max_value)
            del hmap[key_with_max_value]
        return res
